src/data/optimism/velodromeLpPools.json

```diff
@@ -1,4 +1,24 @@
 [
+  {
+    "name": "velodrome-v2-kuji-weth",
+    "address": "0x1e7c5572B3D5942Cf5dabf61f5Ba3FbC6B65e921",
```

Comment:
definitely not kuji lp

Reply:
f. Thought I tripled checked everything. Fixed